### PR TITLE
ticdc: update storage sink doc

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
update storage sink doc

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?
ref https://github.com/pingcap/tiflow/pull/8954

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/13956
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 1]
```diff
@@ -117,18 +117,14 @@ Data change records are saved to the following path:
...
->
-> - After a DDL operation is performed, the table version is the TSO when the DDL is executed in the upstream TiDB. However, the change of the table version does not mean the change of the table schema. For example, adding a comment to a column does not cause the `schema.json` file content to change.
-> - The changefeed process restarts. The table version is the checkpoint TSO when the process restarts. When there are many tables and the process restarts, it takes a long time to traverse all directories and find the position where each table was written last time. Therefore, data is written to a new directory with the version being the checkpoint TSO, instead of to the earlier directory.
-> - After a table scheduling occurs, the table version is the Changefeed checkpoint TSO when the table is scheduled to the current node.
+> The table version changes only after a DDL operation is performed for the upstream table: the table version is the TSO when the upstream TiDB completes the execution of the DDL. However, the change of the table version does not mean the change of the table schema. For example, adding a comment to a column does not cause the schema file content to changed.
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-cloud-storage.md:120]:
```suggestion
> The table version changes only after a DDL operation is performed on the upstream table, and the new table version is the TSO when the upstream TiDB completes the execution of the DDL. However, the change of the table version does not mean the change of the table schema. For example, adding a comment to a column does not cause the schema file content to change.
```

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 2]
```diff
@@ -159,23 +155,27 @@ Metadata is a JSON-formatted file, for example:
...
-```
+- Changes to a new path to write data change records. For example, when the version of `test.table1` changes to `441349361156227074`, TiCDC changes to the `s3://bucket/bbb/ccc/test/table1/441349361156227074/2022-01-02/` the path to write data change records.
+- Generate a schema file in the following path to store the table schema information:
+
+    ```shell
+    {scheme}://{prefix}/{schema}/{table}/meta/schema_{table-version}_{hash}.json
+    ```
 
-The following is a `schema.json` file:
+Taking `schema_441349361156227074_3131721815.json` as an example, the content in the table schema information file is as follows:
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-cloud-storage.md:169]:
```suggestion
Taking the `schema_441349361156227074_3131721815.json` schema file as an example, the table schema information in this file is as follows:
```

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 3]
```diff
@@ -222,9 +223,32 @@ The following is a `schema.json` file:
...
 
+### DDL events at the database level
+
+When a database-level DDL event is performed in the upstream database, TiCDC automatically generates a schema file in the following path to store the database schema information:
+
+```shell
+{scheme}://{prefix}/{schema}/meta/schema_{table-version}_{hash}.json
+```
+
+Taking `schema_441349361156227000_3131721815.json` as an example, the content of the database schema information file is as follows:
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-cloud-storage.md:234]:
```suggestion
Taking the `schema_441349361156227000_3131721815.json` schema file as an example, the database schema information in this file is as follows:
```

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 4]
```diff
@@ -222,9 +223,32 @@ The following is a `schema.json` file:
...
+  "Type": 1,
+  "TableColumns": null,
+  "TableColumnsTotal": 0
+}
+```
+
 ### Data type
 
-This section describes the data types used in the `schema.json` file. The data types are defined as `T(M[, D])`. For details, see [Data Types](/data-type-overview.md).
+This section describes the data types used in the `schema_{table-version}_{hash}.json` file (shorten as schema file). The data types are defined as `T(M[, D])`. For details, see [Data Types](/data-type-overview.md).
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-cloud-storage.md:251]:
```suggestion
This section describes the data types used in the `schema_{table-version}_{hash}.json` file (shorten as schema file in the following sections). The data types are defined as `T(M[, D])`. For details, see [Data Types](/data-type-overview.md).
```

**Review by CharlesCheung96 — APPROVED** [review]

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 5]
```diff
@@ -159,23 +155,27 @@ Metadata is a JSON-formatted file, for example:
...
+### DDL events at the table level
 
-Table schema information is saved in the following path:
+When a DDL event of an upstream table causes a table version change, TiCDC automatically does the following:
 
-```shell
-{scheme}://{prefix}/{schema}/{table}/{table-version-separator}/schema.json
-```
+- Changes to a new path to write data change records. For example, when the version of `test.table1` changes to `441349361156227074`, TiCDC changes to the `s3://bucket/bbb/ccc/test/table1/441349361156227074/2022-01-02/` the path to write data change records.
+- Generate a schema file in the following path to store the table schema information:
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-cloud-storage.md:163]:
```suggestion
- Generates a schema file in the following path to store the table schema information:
```

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 6]
```diff
@@ -222,9 +223,32 @@ The following is a `schema.json` file:
...
 
+### DDL events at the database level
+
+When a database-level DDL event is performed in the upstream database, TiCDC automatically generates a schema file in the following path to store the database schema information:
+
+```shell
+{scheme}://{prefix}/{schema}/meta/schema_{table-version}_{hash}.json
+```
+
+Taking the `schema_441349361156227000_3131721815.json` schema file as an example, the database schema information in this file is as follows:
```
Comment (hfxsd) [inline ticdc/ticdc-sink-to-cloud-storage.md:234]: 中文版也需要改下，schema 改为 schema1
```suggestion
Taking the `schema1_441349361156227000_3131721815.json` schema file as an example, the database schema information in this file is as follows:
```

File: ticdc/ticdc-sink-to-cloud-storage.md [hunk 7]
```diff
@@ -222,9 +223,32 @@ The following is a `schema.json` file:
...
+  "Type": 1,
+  "TableColumns": null,
+  "TableColumnsTotal": 0
+}
+```
+
 ### Data type
 
-This section describes the data types used in the `schema.json` file. The data types are defined as `T(M[, D])`. For details, see [Data Types](/data-type-overview.md).
+This section describes the data types used in the `schema_{table-version}_{hash}.json` file (shorten as schema file in the following sections). The data types are defined as `T(M[, D])`. For details, see [Data Types](/data-type-overview.md).
```
Comment (hfxsd) [inline ticdc/ticdc-sink-to-cloud-storage.md:251]:
```suggestion
This section describes the data types used in the `schema_{table-version}_{hash}.json` file (hereafter referred to as "schema file" in the following sections). The data types are defined as `T(M[, D])`. For details, see [Data Types](/data-type-overview.md).
```

**Review by hfxsd — APPROVED** [review]

File: ticdc/ticdc-sink-to-cloud-storage.md (hunk 6 above)
Comment (CharlesCheung96) [reply ticdc/ticdc-sink-to-cloud-storage.md:234]: 为什么要改为 schema1 呢？

Comment (CharlesCheung96) [reply ticdc/ticdc-sink-to-cloud-storage.md:234]: img_v2_94526e93-bb3d-4ec2-be50-6d073c9d622g
文件名与文件内容无关的

**Review by hfxsd — APPROVED** [review]

**Pull request merged**